README.rst

```diff
@@ -187,7 +187,12 @@ In e.g. your ``init.lua``:
 
         -- Lean 3
         lsp3 = {
-            on_attach = on_attach
+            on_attach = on_attach,
+
+            -- Additional options not passed directly to lspconfig
+            --
+            -- Extra CLI arguments to pass to `lean-language-server`
+            extra_argv = { "-M", "4096" }
```

Comment:
```suggestion
            -- This is the default if unspecified.
            cmd = { "lean-language-server", "--stdio", "-M", "4096" }
```
IMO probably best to avoid introducing new fields to the config passed directly to `nvim-lspconfig` if possible (see below).